src/field/captcha/captcha_pane.jsx

```diff
@@ -7,46 +7,54 @@ import * as l from '../../core/index';
 import { swap, updateEntity } from '../../store/index';
 import * as captchaField from '../captcha';
 import { getFieldValue, isFieldVisiblyInvalid } from '../index';
-
-const Captcha = ({ lock, i18n, onReload }) => {
-  const lockId = l.id(lock);
-
-  function handleChange(e) {
-    swap(updateEntity, 'lock', lockId, captchaField.set, e.target.value);
+import RecaptchaV2 from './recaptchav2';
+
+export default class CaptchaPane extends React.Component {
+  render() {
+    const { i18n, lock, onReload } = this.props;
+
+    const lockId = l.id(lock);
```

Comment:
using a non-descriptive variable `l` makes this hard to understand. perhaps `Lock`?

Reply:
I don't think this will change.. the `l` var has been part of lock SDK for months now, and as far as I know, refers to the context. cc @stevehobbsdev

Reply:
This is literally all over the codebase and is pretty much a convention with this library, has been for years.

I'm not saying it's right and I agree with the review, but this should ideally be changed everywhere. However, I'd suggest it's outside of the scope of this PR.